src/test/java/uk/gov/pay/api/it/CreateAgreementIT.java

```diff
@@ -39,8 +39,7 @@ public void shouldCreateAgreement() {
         postAgreementRequest(agreementPayload(createAgreementRequestParams))
                 .statusCode(HttpStatus.SC_CREATED)
                 .contentType(JSON)
-                .body("agreement_id", is(VALID_AGREEMENT_ID))
-                .body("reference", is(REFERENCE));
+                .body("agreement_id", is(VALID_AGREEMENT_ID));
```

Comment:
Why was reference removed?

Reply:
Yeah good question. The idea will be that Connector returns just the resource ID of the thing it has created, the API can then look this up from the event store and respond.

That idea took a bit of time to develop and for some reason the agreement id and reference were parsed from Connector.